lib/publications/patch-draft.tsx

```diff
@@ -0,0 +1,15 @@
+import { fetchWithSession, Method } from '@/lib/fetch-with-refresh';
+
+export async function pathDraft(formData: FormData, id: string) {
```

Comment:
Pense c'est une typo?
```suggestion
export async function patchDraft(formData: FormData, id: string) {
```

Reply:
Ouais bien vu oups